gloo/docs/installation/gateway/docker-compose-consul/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Run Gloo Gateway Locally with Hashicorp Consul & Vault
+menuTitle: Run Gloo with Consul & Vault
+weight: 5
+description: How to run Gloo Locally using Docker-Compose
+---
+
+## Motivation
+
+While Kubernetes provides APIs for config storage ([CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)), credential storage ([Secrets](https://kubernetes.io/docs/concepts/configuration/secret/)), and service discovery ([Services](https://kubernetes.io/docs/concepts/services-networking/service/)), users may wish to run Gloo without using Kubernetes.
+
+Gloo provides alternate mechanisms for configuration, credential storage, and service discovery that do not require Kubernetes, including the use of local `.yaml` files, [Consul Key-Value storage](https://www.consul.io/api/kv.html) and [Vault Key-Value storage](https://www.vaultproject.io/docs/secrets/kv/kv-v2.html).
+
+This tutorial provides a basic installation flow for running Gloo with 
+Docker Compose, connecting it to Consul for configuration storage and Vault for credential storage.
+
+> Note: the deployment steps in this tutorial should be modified for production environments. This tutorial is  meant as an example of how to configure Gloo to connect to Consul and Vault for configuration/secrets. In a production environment, additional steps should be taken to ensure Gloo has the proper ACL tokens to communicate with production Consul and Vault clusters.
```

Comment:
If you put notes between these two delimiters
```
{{% notice note %}}
<content here>
{{% /notice %}}
```
you get a nice blue box that is in line with line with what we have on other pages.